### ISSUE TITLE
Frsky X18s on Ethos 1.4.14 "audio" option on wireless menu

### ISSUE THREAD

**Issue by Eduardoesteban1694** [FrSkyRC/ETHOS-Feedback-Community#3109]
There is audio option on bluetooth modes, but it doesn't work. I think it should be removed or hidden

**Comment by Eduardoesteban1694** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/152110218/f30e859e-2fd1-4d7c-9ad7-9f04e2fd6c3c)

**Comment by mawzthefinn** [comment]
Yes, that's for the X20S/Pro and XE with BT Audio module not the X18S.

**Comment by salibaj2** [comment]
> Yes, that's for the X20S/Pro and XE with BT Audio module not the X18S.
> 
> Looks like it's detecting on the 'S' model only, my X18 lacks that menu option.

I have an X18SE, if I add the Bluetooth module, the audio option will work?

I think there is already space on the PCB for that audio module.

**Comment by Eduardoesteban1694** [comment]
Maybe it could work, but it's a function that I don't see as being very useful.

**Comment by bsongis-frsky** [comment]
Bluetooth audio support removed from X18S and TWXLiteS
Gyro removed from X18 and TWXLite

**Comment by giaffone** [comment]
> Bluetooth audio support removed from X18S and TWXLiteS Gyro removed from X18 and TWXLite

Hi @bsongis-frsky 
I have an early X18 (non S version) that has working gyro capabilities and I use the gyro function.
Could you not remove it from X18? 

https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/88498987/e8357824-45a7-4ea7-ae65-2b135377a030

**Comment by bsongis-frsky** [comment]
You will have to flash the X18S bootloader / firmware

**Comment by giaffone** [comment]
Hi @bsongis-frsky,
Thanks! Didn't know I could do that.
I presume won't be able to use the Ethos Suite and do future updates manually right?
I'm fine with that.

**Comment by mawzthefinn** [comment]
Once you have the new firmware and bootloader flashed, you should be able to use Suite, it detects based on the versions flashed.

**Comment by giaffone** [comment]
Hi @mawzthefinn,
Flashed the X18S Bootloader and it worked like a charm. The Ethos Suite sees my X18 as an X18S now.
Thanks!!
